<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleWidget</class>
 <widget class="QWidget" name="PeopleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="qss_tags" stdset="0">
   <stringlist notr="true">
    <string>xlet_frame</string>
   </stringlist>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="header_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Contacts</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="qss_tags" stdset="0">
        <stringlist notr="true">
         <string>xlet_h1</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HorizontalMenu" name="menu" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="body_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLineEdit" name="entry_filter">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>search</string>
         </property>
         <property name="qss_tags" stdset="0">
          <stringlist notr="true">
           <string>styled</string>
          </stringlist>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="PeopleEntryView" name="entry_table"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>11</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QPushButton" name="new_contact_button">
         <property name="text">
          <string>NEW CONTACT</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="qss_tags" stdset="0">
          <stringlist notr="true">
           <string>orange</string>
           <string>styled</string>
          </stringlist>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="import_button">
         <property name="text">
          <string>IMPORT</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="qss_tags" stdset="0">
          <stringlist notr="true">
           <string>orange</string>
           <string>styled</string>
          </stringlist>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="export_button">
         <property name="text">
          <string>EXPORT</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="qss_tags" stdset="0">
          <stringlist notr="true">
           <string>orange</string>
           <string>styled</string>
          </stringlist>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="purge_contacts_button">
         <property name="text">
          <string>DELETE ALL CONTACTS</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="qss_tags" stdset="0">
          <stringlist notr="true">
           <string>orange</string>
           <string>styled</string>
          </stringlist>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="status_icon">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>90</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HorizontalMenu</class>
   <extends>QWidget</extends>
   <header location="global">xletlib/horizontal_menu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PeopleEntryView</class>
   <extends>QTableView</extends>
   <header location="global">xlets/people/people_entry_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../xivoclient.qrc"/>
 </resources>
 <connections/>
</ui>
